examples/lax_root_batch.py

```diff
@@ -8,38 +16,88 @@
 
 import root_pandas
 
+
 parser = argparse.ArgumentParser(description="Create lichen ROOT files with lax")
 
 parser.add_argument('--run_number', dest='RUN_NUMBER',
                     action='store', required=True, type=int,
-                    help='Run number to process')
+                    help='Run number to process (-1 for MC)')
```

Comment:
Agree.  But until we get run dependence working at G4/emission/fax/pax level, this doesn't matter.